compiler/x/i386/codegen/OMRTreeEvaluator.cpp

```diff
@@ -3203,6 +3203,11 @@ TR::Register *OMR::X86::I386::TreeEvaluator::lloadEvaluator(TR::Node *node, TR::
    return reg;
    }
 
+TR::Register *OMR::X86::I386::TreeEvaluator::lReadBarrierEvaluator(TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
This evaluator may go into common x86 code, it is same as the 64-bit one.

Reply:
changed according to the comments.